### PR TITLE
[OT149-26] Handle access attempt with invalid role

### PR DESCRIPTION
**Pull request by adriangh1321**
## Summary
- Added control error when accessing with invalid token or role.
- Created CustomAccessDeniedHandler class
- Created CustomAuthenticationEntryPoint class

## Jira ticket
COMO usuario autenticado
QUIERO restringir el acceso a cierto contenido
PARA mantener los datos protegidos 

## Criterios de aceptación:

Deberá chequear si la petición corresponde al usuario, caso contrario deberá devolver un código de error 403 con un mensaje de error.
### Type of change
- [x] New feature
- [ ] New Tests
- [ ] Bug fix
- [ ] Refactor

### Checklist

- [ ] Traceability between this change and Jira.
- [ ] ```mvn clean install``` was run and all tests completed successfully.
- [ ] There are no unused variables and/or imports in the modified classes.
- [ ] There are no imports in the modified classes with the wildcard character. Ex: ```com.somepackage.*```.
- [ ] Slf4j was used for the application log instead ```System.out```.
- [ ] Public methods are documented with ```javadoc```.

### CODE REVIEW

File: src/main/java/com/alkemy/ong/configuration/CustomAccessDeniedHandler.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package com.alkemy.ong.configuration;
```
Comment (manerajona) [inline src/main/java/com/alkemy/ong/configuration/CustomAccessDeniedHandler.java:1]: Could you please move this class to `com.alkemy.ong.security `

File: src/main/java/com/alkemy/ong/configuration/CustomAuthenticationEntryPoint.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package com.alkemy.ong.configuration;
```
Comment (manerajona) [inline src/main/java/com/alkemy/ong/configuration/CustomAuthenticationEntryPoint.java:1]: Could you please move this class to `com.alkemy.ong.security `

Comment (manerajona) [inline src/main/java/com/alkemy/ong/configuration/CustomAuthenticationEntryPoint.java:1]: Could you please move this class to `com.alkemy.ong.security `

**Review by manerajona — CHANGES_REQUESTED** [review]

File: src/main/java/com/alkemy/ong/configuration/CustomAccessDeniedHandler.java (hunk 1 above)
Comment (adriangh1321) [reply src/main/java/com/alkemy/ong/configuration/CustomAccessDeniedHandler.java]: Corregido

File: src/main/java/com/alkemy/ong/configuration/CustomAuthenticationEntryPoint.java (hunk 2 above)
Comment (adriangh1321) [reply src/main/java/com/alkemy/ong/configuration/CustomAuthenticationEntryPoint.java]: Corregido

Comment (adriangh1321) [reply src/main/java/com/alkemy/ong/configuration/CustomAuthenticationEntryPoint.java]: Corregido

**Pull request merged**